### PR TITLE
Added support for PS4 Joystick

### PR DESCRIPTION
**Pull request by TrippleBender**
### Description

Please explain the changes you made, including a reference to the related issue if applicable

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [x] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by TrippleBender** [comment]
I added support for the PS4 Joystick

File: doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst [hunk 1]
```diff
@@ -33,16 +33,18 @@ The script defaults to using ``/dev/input/js0`` for your game controller port. T
...
 This script can read four types of joysticks:
 
-1. XBox360 Controller via USB
-2. PS3 Controller via USB
-3. PS3 Controller via Bluetooth (Please use ps3joy package at `http://wiki.ros.org/ps3joy <http://wiki.ros.org/ps3joy>`_)
-4. Arctic USB Wireless `Gamepad <https://www.arctic.ac/eu_en/usb-wireless-gamepad.html>`_
+1. PS3 Controller via USB
+2. PS3 Controller via Bluetooth (Please use ps3joy package at `http://wiki.ros.org/ps3joy <http://wiki.ros.org/ps3joy>`_)
+3. PS4 Controller via USB
+4. PS4 Controller via Bleutooth (Please use ds4_driver package at `http://wiki.ros.org/ds4_driver <http://wiki.ros.org/ds4_driver>`_)
```
Comment (v4hn) [inline doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst:39]: Typo `Bleutooth`

**Comment by v4hn** [comment]
Depends on https://github.com/ros-planning/moveit/pull/2060

File: doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst [hunk 2]
```diff
@@ -33,16 +33,18 @@ The script defaults to using ``/dev/input/js0`` for your game controller port. T
...
+4. PS4 Controller via Bluetooth (Please use ds4_driver package at `http://wiki.ros.org/ds4_driver <http://wiki.ros.org/ds4_driver>`_) 
+5. XBox360 Controller via USB
+6. Arctic USB Wireless `Gamepad <https://www.arctic.ac/eu_en/usb-wireless-gamepad.html>`_
 
 Joystick Command Mappings
 -------------------------
 
 =====================   ==================   ===================== ==================
-Command                 PS3 Controller       Xbox Controller       Arctic Controller
+Command                 PS3/4 Controller       Xbox Controller       Arctic Controller
```
Comment (v4hn) [inline doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst:47]:
```suggestion
Command                 PS3/4 Controller     Xbox Controller       Arctic Controller
```

File: doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst [hunk 3]
```diff
@@ -33,16 +33,18 @@ The script defaults to using ``/dev/input/js0`` for your game controller port. T
 
 This script can read four types of joysticks:
```
Comment (rhaschke) [inline doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst:34]:
```suggestion
This script can handle the following types of joysticks:
```

**Pull request merged**